artifactory/commands/transferfiles/utils.go

```diff
@@ -264,13 +267,22 @@ func getErrorsFiles(repoKey string, isRetry bool) (filesPaths []string, err erro
 		return []string{}, err
 	}
 
-	filesNames, err := biUtils.ListFiles(dirPath, false)
+	errorsFilesRegex := fmt.Sprintf(errorsFilesRegexFormat, repoKey)
+
+	regExp, err := regexp.Compile(errorsFilesRegex)
+	if errorutils.CheckError(err) != nil {
+		return nil, err
+	}
+
+	files, err := biUtils.ListFiles(dirPath, false)
 	if err != nil {
 		return nil, err
 	}
 
-	for _, file := range filesNames {
-		if strings.HasPrefix(filepath.Base(file), repoKey) {
+	for _, file := range files {
+		matches := regExp.FindStringSubmatch(filepath.Base(file))
+		// Expecting a match and 4 groups.
+		if len(matches) == 5 {
```

Comment:
We're checking that the len is 5, but the above comment says `4 groups`. Is this a mistake?

Reply:
No, this function returns the whole match and the value of each group. I'll try rephrasing the comment above.